ccd-definition/CaseEventToFields/draftCMO.json

```diff
@@ -10,5 +10,41 @@
     "PageLabel": " ",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "orderBasis",
+    "PageFieldDisplayOrder": 2,
```

Comment:
pageFieldDisplayOrder should be 1 -4 here

Reply:
done :)